### ISSUE TITLE
custom dataset

### ISSUE THREAD

**Issue by JavierClearImageAI** [ultralytics/JSON2YOLO#10]
Does it work for a custom coco2Json data-set? When I run it, it returns empty directories of images and labels. I am wondering whether it works only for the 81 to 91 classes of the standard coco data-set.

I am going to keep working with the other popular [convert2Yolo repo](https://github.com/SsaRu/convert2Yolo) for the moment, although this one also has some formatting deficiencies.

Thanks!

**Comment by glenn-jocher** [comment]
@JavierClearImageAI yes, it definitely works with custom COCO-format datasets. Ensure your custom dataset's JSON follows the COCO format with 'images' and 'annotations' keys and use a root 'images' directory containing all images. The "labels" directory is initially empty and will only be populated after training commences and annotations are converted. Also, make sure to specify the correct dataset type in your data.yaml file and ensure all images have corresponding annotations in the JSON. Good luck with your dataset, and we're always here to assist with any issues!

**Comment by glenn-jocher** [comment]
Thanks @JavierClearImageAI. Custom datasets should be organized with image files and matching YOLO txt labels, plus a data.yaml. The updated README now points to current Ultralytics dataset docs and converter examples.
